manim/utils/file_ops.py

```diff
@@ -23,6 +25,12 @@ def add_extension_if_not_present(file_name, extension):
         return file_name
 
 
+def add_version_before_extension(file_name):
+    index = str(file_name).index(Path(file_name).suffix)
+    file_name = str(file_name)
+    return Path(f"{file_name[:index]} - ManimCE v{__version__}{file_name[index:]}")
```

Comment:
This should remove unnecessary conversions and also give names to things which makes the code easier to read imo (it also removes the spaces in the name)
```suggestion
    file_name = Path(file_name)
    name, suffix = file_name.stem, file_name.suffix

    return Path(f"{name}_ManimCE_v{__version__}{suffix}")
```

Reply:
I had to add in the path to the file, but I like it otherwise. I changed it.